lib/Remotery.c

```diff
@@ -4309,6 +4311,17 @@ static void Server_Update(Server* server)
 
 #define SAMPLE_NAME_LEN 128
 
+typedef struct StatInfo
+{
+    enum rmtStatType type;
+    union {
+        rmtI32 ivalue;
+        rmtF32 fvalue;
+    } value;
+    rmtU32 flags;
+    rmtU32 desc; // The hash of the description
```

Comment:
Code style Q: Do you prefer shorter names, or the names spelled out?
E.g.. `desc` vs `description` ?
`StatInfo`vs `StatisticInfo` ?

Reply:
In the implementation file try to be as verbose as you can.